packages/ui/src/ui-component/cards/ItemCard.jsx

```diff
@@ -72,7 +72,7 @@ const ItemCard = ({ data, images, onClick }) => {
                                         flexShrink: 0,
                                         marginRight: 10,
                                         borderRadius: '50%',
-                                        background: data.color
+                                        backgroundImage: data.color
```

Comment:
this should still be `background` no?

Reply:
I did this to make the code consistent,when their is no image then also instead of background i changed the name to backgroundimage

Reply:
but `backgroundImage` is only when we have an image url. For this case, its just a color, we should use `background`